cmd/kubebuilder/v0/version0.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
We can probably rename this file to `commands.go` ?  same for the version1.go under v1 dir.